app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala

```diff
@@ -34,7 +34,7 @@ final case class CompleteReturn(
   reliefDetails: CompleteReliefDetailsAnswers,
   exemptionsAndLossesDetails: CompleteExemptionAndLossesAnswers,
   yearToDateLiabilityAnswers: CompleteCalculatedYearToDateLiabilityAnswers,
-  initialGainOrLoss: Option[AmountInPence]
+  initialGainOrLoss: Option[AmountInPenceWithSource]
```

Comment:
this should just be `AmountInPence` - it will always be a user supplied value

Reply:
Done